pyart/io/nexrad_level2.py

```diff
@@ -176,9 +178,15 @@ def __init__(self, filename):
 
         # pull out the vcp record
         msg_5 = [r for r in self._records if r['header']['type'] == 5]
+
+        wstring1 = 'No MSG5 detected. Setting to meaningless data \n'
```

Comment:
@scollis I would maybe remove these lines and added the message within the warnings.warn, see below.